### PR TITLE
Completed for review

### PR DESCRIPTION
**Pull request by Abhimanyu1311**

### CODE REVIEW

File: src/Components/addToCart.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+const addToCart = (id, image, name, price,quantity) => {
+    const cart = JSON.parse(localStorage.getItem('cart')) || [];
+    const cartItem = cart.find(item => item.id === id);
+    if (cartItem) {
+      cartItem.quantity += 1;
+    } else {
+      cart.push({ id, name, image, price, quantity: 1 });
+    }
+    localStorage.setItem('cart', JSON.stringify(cart));
+    console.log(cart);
```
Comment (ankur1778) [inline src/Components/addToCart.js]: remove logs

File: src/Pages/AddressPage.js [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+
+    const states = [
+        "Andhra Pradesh", "Arunachal Pradesh", "Assam", "Bihar", "Chhattisgarh", "Goa", "Gujarat", "Haryana", "Himachal Pradesh", "Jammu and Kashmir",
+        "Jharkhand", "Karnataka", "Kerala", "Madhya Pradesh", "Maharashtra", "Manipur", "Meghalaya", "Mizoram", "Nagaland", "Odisha", "Punjab", "Rajasthan",
+        "Sikkim", "Tamil Nadu", "Telangana", "Tripura", "Uttarakhand", "Uttar Pradesh", "West Bengal", "Andaman and Nicobar Islands",
+        "Chandigarh", "Dadra and Nagar Haveli", "Daman and Diu", "Delhi", "Lakshadweep", "Puducherry"
+    ]
+
+    const handleAddress = async (values, { resetForm }) => {
+        console.log(values);
```
Comment (ankur1778) [inline src/Pages/AddressPage.js]: remove logs

File: src/Pages/Login.js [hunk 3]
```diff
@@ -37,62 +38,74 @@ export default function Login() {
...
-                </Formik>
-            </div>
-        </div>
+                                </Form>
+                            )}
+                        </Formik>
+                    </div>
+                </div>
+                :
+                <div>HIIIIIIII</div>
```
Comment (ankur1778) [inline src/Pages/Login.js]: Please explain this or remove this

File: src/Components/addToCart.js (hunk 1 above)
Comment (Abhimanyu1311) [reply src/Components/addToCart.js]: removed

File: src/Pages/Login.js (hunk 3 above)
Comment (Abhimanyu1311) [reply src/Pages/Login.js]: removed

**Pull request merged**